bfs_test.go

```diff
@@ -1,14 +1,23 @@
 package bfs_test
 
 import (
+	"context"
+	"net/url"
 	"testing"
 
+	"github.com/bsm/bfs"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 // ------------------------------------------------------------------------
 
+func init() {
+	bfs.Register("mem", func(_ context.Context, u *url.URL) (bfs.Bucket, error) {
+		return bfs.NewInMem(), nil
```

Comment:
Well, in this case the scheme works more like a `null://` or `blackhole://` :)

I'd suggest to have a global map of named in-mem storage instances, but doesn't look that it'll lay well onto current version (mean godoc comments from https://github.com/bsm/bfs/pull/13 )

Reply:
Oh, sorry, ignore me - it's in `_test.go`.